sentinel/memstore.go

```diff
@@ -89,18 +89,18 @@ func (k *MemStore) fetchContract(key string) (types.Contract, error) {
-		Chain          common.Chain       `protobuf:"varint,2,opt,name=chain,proto3,casttype=github.com/arkeonetwork/arkeo/common.Chain" json:"chain,omitempty"`
-		Client         common.PubKey      `protobuf:"bytes,3,opt,name=client,proto3,casttype=github.com/arkeonetwork/arkeo/common.PubKey" json:"client,omitempty"`
-		Delegate       common.PubKey      `protobuf:"bytes,4,opt,name=delegate,proto3,casttype=github.com/arkeonetwork/arkeo/common.PubKey" json:"delegate,omitempty"`
-		Type           types.ContractType `protobuf:"varint,5,opt,name=type,proto3,enum=arkeo.arkeo.ContractType" json:"type,omitempty"`
-		Height         string             `protobuf:"varint,6,opt,name=height,proto3" json:"height,omitempty"`
-		Duration       string             `protobuf:"varint,7,opt,name=duration,proto3" json:"duration,omitempty"`
-		Rate           string             `protobuf:"varint,8,opt,name=rate,proto3" json:"rate,omitempty"`
-		Deposit        string             `protobuf:"varint,9,opt,name=deposit,proto3" json:"deposit,omitempty"`
-		Paid           string             `protobuf:"varint,10,opt,name=paid,proto3" json:"paid,omitempty"`
-		Nonce          string             `protobuf:"varint,11,opt,name=nonce,proto3" json:"nonce,omitempty"`
-		ClosedHeight   string             `protobuf:"varint,12,opt,name=closed_height,json=closedHeight,proto3" json:"closed_height,omitempty"`
+		ProviderPubKey   common.PubKey      `protobuf:"bytes,1,opt,name=provider_pub_key,json=providerPubKey,proto3,casttype=github.com/arkeonetwork/arkeo/common.PubKey" json:"provider_pub_key,omitempty"`
+		Chain            common.Chain       `protobuf:"varint,2,opt,name=chain,proto3,casttype=github.com/arkeonetwork/arkeo/common.Chain" json:"chain,omitempty"`
+		Client           common.PubKey      `protobuf:"bytes,3,opt,name=client,proto3,casttype=github.com/arkeonetwork/arkeo/common.PubKey" json:"client,omitempty"`
+		Delegate         common.PubKey      `protobuf:"bytes,4,opt,name=delegate,proto3,casttype=github.com/arkeonetwork/arkeo/common.PubKey" json:"delegate,omitempty"`
+		Type             types.ContractType `protobuf:"varint,5,opt,name=type,proto3,enum=arkeo.arkeo.ContractType" json:"type,omitempty"`
+		Height           string             `protobuf:"varint,6,opt,name=height,proto3" json:"height,omitempty"`
+		Duration         string             `protobuf:"varint,7,opt,name=duration,proto3" json:"duration,omitempty"`
+		Rate             string             `protobuf:"varint,8,opt,name=rate,proto3" json:"rate,omitempty"`
+		Deposit          string             `protobuf:"varint,9,opt,name=deposit,proto3" json:"deposit,omitempty"`
+		Paid             string             `protobuf:"varint,10,opt,name=paid,proto3" json:"paid,omitempty"`
+		Nonce            string             `protobuf:"varint,11,opt,name=nonce,proto3" json:"nonce,omitempty"`
+		SettlementHeight string             `protobuf:"varint,12,opt,name=settlement_height,json=SettlementHeight,proto3" json:"settlement_height,omitempty"`
```

Comment:
should be lower-case `json=settlementHeight,`?

Reply:
fixed.